pom.xml

```diff
@@ -21,7 +21,6 @@
   ~ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
   ~ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
   ~ SOFTWARE.
-  ~
```

Comment:
This is just an "empty" line removable, but do you agree to this @dheerajodha, as this file was added by you?

Reply:
I agree.